### PR TITLE
Created question helpfulness button that makes call to API

### PR DESCRIPTION
**Pull request by davidlichter**
Also created a title with a small logo to replace the standard react icon

### CODE REVIEW

File: src/reducers/QAReducers.js [hunk 1]
```diff
@@ -90,3 +90,15 @@ export const putHelpfulness = (answerID) => {
       console.log(error);
     });
 };
+
+export const putHelpfulnessQuestion = (questionID) => {
+  fetch(`http://3.21.164.220/qa/questions/${questionID}/helpful`, {
+    method: 'PUT',
+  })
+    .then((res) => {
+      console.log('Feedback Received!');
```
Comment (jessepye) [inline src/reducers/QAReducers.js]: One random console.log that could be removed

**Review by jessepye — CHANGES_REQUESTED** [review]
Looks good! Found one stray console.log that could be removed.

File: src/reducers/QAReducers.js (hunk 1 above)
Comment (davidlichter) [reply src/reducers/QAReducers.js]: @jessepye I removed the console logs. They only would run when clicking 'helpful', but they were unneccesary

**Review by jessepye — APPROVED** [review]
lookin good

**Pull request merged**